spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1439,6 +1451,13 @@ A client for the Spring Cloud Stream schema registry can be configured using the
   }
 ----
 
+[NOTE]
+====
+The default client caches all responses locally to avoid unnecessary network calls. It leverages Spring caching abstraction, so you can provide
+an external cache provider such as eh-cache or hazelcast if you want to have distributed cache control and invalidation. Schemas are append only entities
+but deletes may occur, make sure you configure your external cache provider to invalidate the cache from time to time to avoid synchronization issues.
+====
```

Comment:
Can we detail a little more how that works, as well as the names of the two cache regions that we use?